### ISSUE TITLE
Question: background may be slower

### ISSUE THREAD

**Issue by BufferUnderflower** [flutterchina/dio_flutter_transformer#6]
Hello, thanks for this handy package! In Readme you're saying that using background compute can be slower. Could you please share sample data / describe in brief your findings?  
I assume results heavily depend on particular device and parallelism implementation by different vendors.
But may be you can share some rough data just to know what to expect, like worse case scenarios, severity and share of problem cases you observed. 
Thank you!

**Comment by mars3142** [comment]
I did use this implementation (got the project from someone else) and we do at startup 9 rest calls and every of them created an isolate with compute. I measured the app start (with a simple timer) and without this package, the app started 2sec faster, because we don't create any threads.

After that, I switched to [Work Manager](https://pub.dev/packages/worker_manager) and spawn at startup an isolate pool and reuse the isolates for every rest call. This approach is way better, than every time to spawn a new isolate.

I wouldn't recommend this package.

**Comment by espadon8004** [comment]
@mars3142 so if I understand correctly, you reuse the same isolate only to process the "jsonDecode" ?

**Comment by mars3142** [comment]
@espadon8004 Correct. This will do the trick. Use of an isolate pool and reuse them.
